betelgeuse.py

```diff
@@ -574,6 +621,40 @@ def cli(context, jobs):
     testimony.SETTINGS['minimum_tokens'] = ['id']
 
 
+@cli.command('create-plan')
```

Comment:
We need to name this command as `plan` or `test-plan` to follow the other commands name convention.